documentation/src/main/asciidoc/topics/config_examples/hotrod_connector_sasl.xml

```diff
@@ -2,14 +2,17 @@
            xsi:schemaLocation="urn:infinispan:server:{schemaversion}
            https://infinispan.org/schemas/infinispan-server-{schemaversion}.xsd"
            xmlns="urn:infinispan:server:{schemaversion}"
-           socket-binding="default" security-realm="default"> <1>
+           socket-binding="default"
+           security-realm="default">
    <hotrod-connector>
       <authentication>
-         <sasl mechanisms="SCRAM-SHA-512 SCRAM-SHA-384 SCRAM-SHA-256 <2>
+         <!-- Specifies SASL mechanisms to use for authentication. -->
+         <!-- Defines the name that the server declares to clients. -->
```

Comment:
I'm missing the callouts here, as comments are not supported between attributes.
Also the comment for `qop` seems to be missing.

Reply:
@danberindei Yes, I agree that callouts are handy b/c you can't add comments between attributes. One thing I noticed while I was going through all these examples is that we duplicate some of the attribute descriptions that are in the schema. 

I feel like we should get away from doing that, which is why I didn't include a comment for `qop`. I kind of feel like the comments in the config examples should be more about explaining "the basics" to help users get going rather than documenting attributes. Maybe that distinction isn't quite clear and I'm not doing too great a job with it. But, overall, I would prefer to have minimal comments in the examples. Going back and looking at the callouts, some of them were definitely doing more documentation than anything else, which is the job of the schema.